packages/dashboard/src/pages/Wallets/components/CreateWallet/index.tsx

```diff
@@ -83,33 +79,32 @@ const useTabPanelStyles = makeStyles(() => ({
 export const CreateWallet = memo(() => {
     const t = useDashboardI18N()
 
-    const tabs = useTabs(
-        t.wallets_create_wallet_tabs(),
-        {
-            mnemonic: t.wallets_wallet_mnemonic(),
-            json: t.wallets_wallet_json_file(),
-            privateKey: t.wallets_wallet_private_key(),
-        },
-        {
-            mnemonic: (
-                <>
+    const panelStyles = useTabPanelStyles()
+    const [currentTab, onChange, tabs] = useTabs('mnemonic', 'json', 'privateKey')
+    return (
+        <Container>
+            <TabContext value={currentTab}>
+                <ButtonGroupTabList onChange={onChange} aria-label={t.wallets_create_wallet_tabs()}>
+                    <Tab label={t.wallets_wallet_mnemonic()} value={tabs.mnemonic} />
+                    <Tab label={t.wallets_wallet_json_file()} value={tabs.json} />
+                    <Tab label={t.wallets_wallet_private_key()} value={tabs.privateKey} />
+                </ButtonGroupTabList>
+                <TabPanel classes={panelStyles} value={tabs.mnemonic}>
                     <Refresh>
                         <RefreshIcon />
                         <Typography>{t.wallets_create_wallet_refresh()}</Typography>
                     </Refresh>
                     <MnemonicGeneratorContainer>
                         <MnemonicReveal words={[...Array(12).keys()].map((i) => String(i))} />
                     </MnemonicGeneratorContainer>
-                </>
-            ),
-            json: 'TODO: ',
-            privateKey: <PrivateKeyInput />,
-        },
-        { variant: 'buttonGroup', tabPanelClasses: useTabPanelStyles() },
-    )
-    return (
-        <Container>
-            {tabs}
+                </TabPanel>
+                <TabPanel classes={panelStyles} value={tabs.json}>
+                    TODO
```

Comment:
@UncleBill Will do it in the future?

Reply:
It's the import-from-json feature, The main functions have been implemented in #3404. It might be easy to integrate in.